### ISSUE TITLE
Autocomplete improvements for tables

### ISSUE THREAD

**Issue by qwesda** [jakob/Postico#474]
When completing tables (or table like things) the priority of completed items could be improved. 
For one thing it makes no sense to offer columns as a completion after `FROM`, `JOIN` keywords.

I think a good list of options (sorted by priority) would be:

- CTE names
- tables/views that are directly visible, i.e. that belong to the `PATH`
- schemas
- functions

After the `UPDATE` keyword the list would not contain CTEs and functions.

**Comment by jakob** [comment]
Yes, that's a very good point. At the moment, autocomplete is still very naive, and we're evaluating how we can make it more context-aware.

**Comment by qwesda** [comment]
I think. I was wrong with the order of entities in the `FROM`, `JOIN` autocomplete lists.

Schemas should be sorted before table/view names since there typically are less of them.
